### PR TITLE
FP-2447 - Info tab closes when adding an import

### PR DESCRIPTION
**Pull request by PedroCristovao-Movai**
- [FP-2447 - Info tab closes when adding an import](https://movai.atlassian.net/browse/FP-2447)

### CODE REVIEW

File: src/engine/ReactPlugin/ToolReactPlugin.js [hunk 1]
```diff
@@ -24,18 +24,19 @@ export function withToolPlugin(ReactComponent, methods = []) {
...
     useEffect(() => {
       PluginManagerIDE.resetBookmarks();
-      on(PLUGINS.TABS.NAME, PLUGINS.TABS.ON.ACTIVE_TAB_CHANGE, async () => {
-        PluginManagerIDE.resetBookmarks();
+      on(PLUGINS.TABS.NAME, PLUGINS.TABS.ON.ACTIVE_TAB_CHANGE, async ({id}) => {
+        if(profile.name === id) PluginManagerIDE.resetBookmarks();
       });
 
       return () => {
+        console.log(">>> debug toolplugin unsubscribe ACTIVE_TAB_CHANGE")
```
Comment (quirinpa) [inline src/engine/ReactPlugin/ToolReactPlugin.js]: Please remove console.log

File: src/engine/ReactPlugin/ToolReactPlugin.js [hunk 2]
```diff
@@ -24,18 +24,19 @@ export function withToolPlugin(ReactComponent, methods = []) {
...
         off(PLUGINS.TABS.NAME, PLUGINS.TABS.ON.ACTIVE_TAB_CHANGE);
       };
     }, []);
```
Comment (quirinpa) [inline src/engine/ReactPlugin/ToolReactPlugin.js]: maybe add profile.name to dependency array

**Review by quirinpa — CHANGES_REQUESTED** [review]

File: src/engine/ReactPlugin/ToolReactPlugin.js (hunk 2 above)
Comment (PedroCristovao-Movai) [reply src/engine/ReactPlugin/ToolReactPlugin.js]: resolved

File: src/engine/ReactPlugin/ToolReactPlugin.js (hunk 1 above)
Comment (PedroCristovao-Movai) [reply src/engine/ReactPlugin/ToolReactPlugin.js]: resolved

**Review by quirinpa — APPROVED** [review]

**Pull request merged**